src/content/refi/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Regenerative finance (ReFi)
+description: An overview of current use cases of blockchain technology for environmental and social impact.
+lang: en
+template: use-cases
+emoji: ":recycle:"
+sidebarDepth: 2
+image: ../../assets/impact_transparent.png
+alt: ""
+summaryPoint1: An alternative application of cryptoeconomic systems focused on sustainability and addressing societal problems.
+summaryPoint2: A tool to bring unprecedented transparency and drastically scale up current environmental asset markets like the voluntary carbon market.
+summaryPoint3: Attempts to mitigate and/or reduce the impact of global and regional crises.
+---
+
+## The Roots of ReFi: Regenerative Capitalism and the Cypherpunks {#roots-of-refi}
+
+In his [2015 paper](https://capitalinstitute.org/wp-content/uploads/2015/04/2015-Regenerative-Capitalism-4-20-15-final.pdf) Regenerative Capitalism: How Universal Principles and Patterns Will Shape Our New Economy, John Fullerton of the Capital Institute set out an evolved version of capitalism intended to account for and address negative externalities such as climate change and entrenched inequality.
+
+Reflecting on the financial crisis of 2008, Fullerton joined many in questioning the long-term viability of early 21st century capitalism and sought to imagine “a self-organizing, naturally self-maintaining, highly adaptive Regenerative form of capitalism that produces lasting social and economic vitality for global civilization as a whole”.
+
+Drawing on the theory of [biomimicry](https://biomimicry.org/what-is-biomimicry/) and applying systems thinking to economics, Fullerton identified eight interconnected principles that would underlie systemic health:
+
+In parallel to Fullerton’s reforming vision of traditional economics, the Cypherpunk movement, which began in the 1970s, evolved to encompass cryptocurrencies such as Bitcoin and the early smart contracts of [Nick Szabo](https://en.wikipedia.org/wiki/Nick_Szabo). By the late 2010s and early 2020s, Ethereum and other blockchains that facilitate strong cryptography and privacy-enhancing technologies had become a reality, offering the possibility of globally coordinated social and political change across the borders of nation states.

```

Comment:
I'm not sure Ethereum really delivers on "privacy enhancing technologies" yet - TC is sanctioned and we don't have privacy built in at the protocol level. The USP of Ethereum right now, as I see it, is the coordination made possible by permissionless global access to a computer with a universally shared state.